README.md

```diff
@@ -137,10 +137,16 @@ monetized field, you can use the `:allow_nil` parameter:
 monetize :optional_price_cents, allow_nil: true
 
 # in Migration
+# To set only the 'amount' column as null: true
 def change
   add_money :products, :optional_price, amount: { null: true, default: nil }
 end
 
+# in Migration to set both 'amount' and 'currency' columns to null: true
+def change
```

Comment:
Might be a conflict artefact, but this is redefining the `#change` method

Reply:
You are right. It is in the example. It is another example how the change method could be used. It is not redefining as the example does not represent a definition.

I could think about moving it as a new example separate example.